protocol/messenger_communities.go

```diff
@@ -347,6 +347,26 @@ func (m *Messenger) SetMuted(communityID types.HexBytes, muted bool) error {
 	return m.communitiesManager.SetMuted(communityID, muted)
 }
 
+func (m *Messenger) MuteCategory(communityID string, categoryID string, muted bool) error {
```

Comment:
Maybe I would rename this method as something like `SetMutePropertyOnChats` or something similar (can't figure out a good name :) ), `MuteCategory` gave me the impression that you want to mute the chats, but you might effectively unmuting them.